### PR TITLE
Update flux and external-dns to handle blue green deployments.

### PR DESCRIPTION
**Pull request by phillebaba**
This change fixes the owner id in eks-core for external-dns, and the cluster directories for fluxv2. The changes in fluxv2 is required so that two clusters do not attempt to write to the same directory.

### CODE REVIEW

**Review by simongottschlag — DISMISSED** [review]
Looking good @phillebaba! 💥🚀

**Review by simongottschlag — APPROVED** [review]

**Comment by simongottschlag** [comment]
@phillebaba will this be a breaking change?

**Review by NissesSenap — APPROVED** [review]
LGTM

File: modules/aws/eks/outputs.tf [hunk 1]
```diff
@@ -43,3 +49,8 @@ output "xenit_config" {
     role_arn = module.xenit.role_arn
   }
 }
+
+output "aws_eks_core_dns_addon" {
+  description = "Core DNS addon"
+  value       = aws_eks_addon.core_dns
+}
```
Comment (phillebaba) [inline modules/aws/eks/outputs.tf:56]: Should be removed

**Review by NissesSenap — APPROVED** [review]
LGTM

**Review by NissesSenap — APPROVED** [review]

**Pull request merged**